### PR TITLE
add /config API, allowing change of debug setting

### PR DESCRIPTION
**Pull request by sduthil**
Depends-On: https://github.com/wazo-platform/wazo-auth-client/pull/21

### CODE REVIEW

**Comment by sduthil** [comment]
recheck

File: wazo_auth/plugins/http/config/http.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from .schemas import config_patch_schema
+
+
+class ConfigResource(AuthResource):
+    def __init__(self, config_service):
+        self._config_service = config_service
+
+    @required_acl('auth.config.read')
+    def get(self):
+        return self._config_service.get_config(), 200
```
Comment (fblackburn1) [inline wazo_auth/plugins/http/config/http.py:18]: Should be restricted to master tenant
database credential and other sensitive information can be retrieved

**Review by fblackburn1 — COMMENTED** [review]
Just to be sure `top` vs `master` (in other services) I guess it's because wazo-auth doesn't have `master` terminology?

**Review by fblackburn1 — APPROVED** [review]

**Comment by sduthil** [comment]
> Just to be sure `top` vs `master` (in other services) I guess it's because wazo-auth doesn't have `master` terminology?

Yes, exactly.

**Pull request merged**